src/Tribe/RSVP.php

```diff
@@ -581,6 +661,184 @@ public function get_statuses_by_action( $action ) {
 		return $status_mgr->get_statuses_by_action( $action, 'rsvp' );
 	}
 
+	/**
+	 * Create an attendee for a RSVP ticket.
+	 *
+	 * @since TBD
+	 *
+	 * @param \Tribe__Tickets__Ticket_Object $ticket        Ticket object.
+	 * @param array                          $attendee_data Attendee data.
+	 *
+	 * @return int Attendee ID.
+	 *
+	 * @throws Exception
```

Comment:
:no_entry_sign: Comment missing for @throws tag in function comment

<sub>hash: 939967c546cbca84c4e3a2fe34d1db49</sub>